modules/appnexusBidAdapter.js

```diff
@@ -1036,4 +1037,21 @@ function addUserId(eids, id, source, rti) {
   return eids;
 }
 
+function getBidFloor(bid) {
+  if (!utils.isFn(bid.getFloor)) {
+    return (bid.params.reserve) ? bid.params.reserve : null;
+  }
+
+  let floor = bid.getFloor({
+    currency: 'USD',
+    mediaType: '*',
+    size: '*',
+    bidRequest: bid
```

Comment:
you can leave bidRequest out, it gets the context from `this`

Reply:
updated.